### ISSUE TITLE
Coding Standards Enforcement

### ISSUE THREAD

**Issue by ibrahimBeladi** [usernane/webfiori#7]
Use something like https://github.com/FriendsOfPHP/PHP-CS-Fixer

```
php .\php-cs-fixer fix --config=.php_cs.dist
```

```
<?php
$finder = PhpCsFixer\Finder::create()
    ->exclude('php-cs-fixer') // if php-cs-fixer is downloaded manually
    ->in(__DIR__)
    ->append([__DIR__.'/php-cs-fixer']);

$config = PhpCsFixer\Config::create()
    ->setRules([
        'single_line_after_imports' => true,
        'global_namespace_import' => true,
        'no_unused_imports' => true,
        'single_import_per_statement' => true,
        'ordered_imports' => true,
        'phpdoc_var_annotation_correct_order' => true,
        'single_blank_line_at_eof' => true,
        'blank_line_after_namespace' => true,
        'line_ending' => true,
        'phpdoc_trim_consecutive_blank_line_separation' => true,
        'single_blank_line_before_namespace' => true,

    ])
    ->setFinder($finder);

return $config;
```

**Comment by ibrahimBeladi** [comment]
For further rules, please refer to https://mlocati.github.io/php-cs-fixer-configurator

**Comment by usernane** [comment]
Fixed all coding standards on commit [dcb39c52172bd2655f68596f1ad905269e53c677](https://github.com/usernane/webfiori/commit/dcb39c52172bd2655f68596f1ad905269e53c677)
